src/storage.ts

```diff
@@ -754,19 +1052,37 @@ export class Storage extends Service {
       }
     );
   }
+
+  /**
+   * Get a reference to an HmacKey object.
+   * Note: this does not fetch the HMAC key's metadata. Use HmacKey#get() to
+   * retrieve and populate the metadata.
+   *
+   * @param {string} accessId The HMAC key's access ID.
+   * @returns {HmacKey}
+   * @see HmacKey
+   *
+   * @example
+   * const {Storage} = require('@google-cloud/storage');
+   * const storage = new Storage();
+   * const hmacKey = storage.hmacKey('ACCESS_ID');
+   */
+  hmacKey(accessId: string) {
+    return new HmacKey(this, accessId);
+  }
 }
 
 /*! Developer Documentation
  *
  * These methods can be auto-paginated.
  */
-paginator.extend(Storage, 'getBuckets');
+paginator.extend(Storage, ['getBuckets', 'getHmacKeys']);
 
 /*! Developer Documentation
  *
  * All async methods (except for streams) will return a Promise in the event
  * that a callback is omitted.
  */
 promisifyAll(Storage, {
-  exclude: ['bucket', 'channel'],
+  exclude: ['bucket', 'channel', 'hmacKey'],
```

Comment:
I think this needs the test updated.

Reply:
Added to `promisifyAll` test to make sure `hmacKey` is in the array passed to the function. Did I miss another place where tests should be updated?

Reply:
Nope, that was all. Thanks!